### PR TITLE
fix extensibility badges

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- fix displaying badges, bring back old naming convention.

**Related issue(s)**

### CODE REVIEW

**Review by grego952 — DISMISSED** [review]

File: examples/injections/functions.yaml [hunk 1]
```diff
@@ -17,15 +17,15 @@ injections: |-
       - name: header.status
         widget: Badge
         highlights:
-          success:
+          positive:
             - RUNNING
-          error:
+          negative:
```
Comment (chriskari) [inline examples/injections/functions.yaml:22]: shouldn't this one be called "warning" for consistency with the documentation and the other files?

File: public/schemas/schema-details.yaml [hunk 2]
```diff
@@ -119,23 +119,31 @@ $widgets:
...
+            positive:
               type: [string, array]
               widget: MultiType
               widgets:
                 string: Jsonata
                 array: SimpleList
               items:
                 type: string
-            error:
+            negative:
```
Comment (chriskari) [inline public/schemas/schema-details.yaml:138]: "warning" here as well?

**Review by chriskari — APPROVED** [review]

File: docs/extensibility/display-section.md [hunk 3]
```diff
@@ -211,18 +211,18 @@ Badge widgets render texts as a status badge, using a set of predefined rules to
...
 
 When no highlights are provided, the following values are automatically handled:
 
-- rendered as information: `initial`, `pending`, `available`, `released`.
-- rendered as success: `ready`, `bound`, `running`, `success`, `succeeded`, `ok`.
+- rendered as informative: `initial`, `pending`, `available`, `released`.
+- rendered as positive: `ready`, `bound`, `running`, `success`, `succeeded`, `ok`.
 - rendered as warning: `unknown`, `warning`.
-- rendered as error: `error`, `failure`, `invalid`.
+- rendered as critical: `error`, `failure`, `invalid`.
```
Comment (akucharska) [inline docs/extensibility/display-section.md:225]: Change also in examples, line 233 and 837

**Review by akucharska — CHANGES_REQUESTED** [review]
Change also in:

- examples/injections/apirules.yaml
- examples/services/configuration.yaml
- src/components/Extensibility/components/tests/Badge.test.js
- tests/integration/fixtures/examples/injections/apirules.yaml
- tests/integration/fixtures/examples/services/configuration.yaml

**Review by grego952 — APPROVED** [review]

**Pull request merged**